ecommers/lib/ui/widgets/recently_viewed_widget/product_item_small.dart

```diff
@@ -0,0 +1,60 @@
+import 'package:ecommers/ui/decorations/dimens/index.dart';
+import 'package:ecommers/ui/decorations/index.dart';
+import 'package:ecommers/ui/widgets/recently_viewed_widget/index.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter/widgets.dart';
+
+class ProductItemSmall extends StatelessWidget {
```

Comment:
U didn't understand me. I meant that such widget already exists and you don't need to create a new one

Reply:
lib/ui/widgets/product_item/product_item_small.dart

Reply:
sorry)